engines/teenagent/metaengine.cpp

```diff
@@ -22,18 +22,40 @@
 #include "common/system.h"
 #include "common/savefile.h"
 #include "common/algorithm.h"
+#include "common/translation.h"
 
 #include "base/plugins.h"
 
 #include "engines/advancedDetector.h"
 #include "teenagent/resources.h"
 #include "teenagent/teenagent.h"
+#include "teenagent/detection.h"
 #include "graphics/thumbnail.h"
 
 enum {
 	MAX_SAVES = 20
 };
 
+#ifdef USE_TTS
+
+static const ADExtraGuiOptionsMap optionsList[] = {
+	{
+		GAMEOPTION_TTS,
+		{
+			_s("Enable Text to Speech"),
```

Comment:
Since you now added translatable strings, please add `engines/teenagent/metaengine.cpp` to `engines/teenagent/POTFILES`